views/components/certification/Cert_1.js

```diff
@@ -0,0 +1,61 @@
+import React, { Fragment } from 'react'
+import styled from 'styled-components/macro'
+import CertIcon from '~/views/components/certification/CertIcon'
+import Info from '~/views/components/certification/Info'
+import Title from '~/views/components/certification/Title'
+import Item from '~/views/components/certification/Item'
+import { CERT_ACTIVE_ITEM } from '~/consts/certification'
+import { leftInfo } from '~/consts/certification'
+import { zIndexArr } from '~/consts/certification'
+import { leftPos } from '~/consts/certification'
+import PropTypes from 'prop-types'
+import { Translate } from 'react-localize-redux'
+
+const Cert_1 = ({ itemSelected, setSelectedItem, title, content, id }) => {
+  return (
+    <Fragment>
+      <CertItem onClick={() => setSelectedItem(id)}>
+        <CertIcon id={id} itemSelected={itemSelected} name="c" />
+      </CertItem>
+
+      <CertInfo itemSelected={itemSelected}>
+        <Title>
+          <span>{id}</span> &mdash; <Translate id={title} />
+        </Title>
+        <p>
+          <Translate id={content} />
+        </p>
+      </CertInfo>
+    </Fragment>
+  )
+}
+
+export default Cert_1
+
+Cert_1.propTypes = {
+  itemSelected: PropTypes.any,
+  setSelectedItem: PropTypes.func,
+  title: PropTypes.string,
+  content: PropTypes.string,
+  id: PropTypes.number
+}
+
+const CertItem = styled(Item)`
+  ${({ id }) =>
+    id != 1 &&
```

Comment:
![image](https://user-images.githubusercontent.com/7911873/70876067-cd931c00-1fea-11ea-853e-7f23dc45e92f.png)
what are we doing here? 🍡 
Can we just keep it simple as the image?
`
left: ${ ({ id }) => leftPos[id - 1] }; 
z-index: ${({ id }) => zIndexArr[id - 1] };
`

Reply:
for margin-left, same thing
`${({ id }) => id == CERT_ACTIVE_ITEM[0]  &&  'margin-left: 0%;' }`

Reply:
updated